test/CadlRanchProjects/azure/core/basic/src/Generated/tspCodeModel.json

```diff
@@ -121,7 +121,7 @@
      "Description": "The entity tag for this resource.",
      "Type": {
       "$id": "17",
-      "Name": "string",
+      "Name": "eTag",
```

Comment:
I think @m-nash talked about seperating typespec version upgrade with cadl-ranch upgrade.
I heard there is no breaking change in this typespec version, so I would like to confirm, if we didn't do anything on cadl-ranch, will all existing cases still work for new version of the emitter?

Reply:
there are many breaking changes in this typespec version so not sure what this comment means.